test/infrastructure/scripts/step-02-eksctl-cluster.sh

```diff
@@ -1,20 +1,28 @@
-eksctl create cluster -f - << EOF
+cmd="create"
+eksctl get cluster --name "${CLUSTER_NAME}" && cmd="upgrade"
+eksctl ${cmd} cluster -f - << EOF
```

Comment:
Which eksctl version are you using? Maybe we should pin it. The Tekton tasks use v0.86.0 but that can't create 1.22 clusters.

Reply:
why are we using an old eksctl version?

Reply:
but either way this is the host cluster not spun up w/ tekton so should be fine. I'm using v0.94.0 and the latest version is v0.103.0

Reply:
I use v0.86.0 locally because I've had issues before when I upgrade about different assumptions made. I'm happy to pin to the latest version possible if it works.